cluster/manifests/ci/base/templates/build-container-template.yaml

```diff
@@ -34,6 +34,28 @@ spec:
 
         SERVICES=$(builder --changed --status=pending --sha={{inputs.parameters.sha}})
         echo $SERVICES
+  - name: changed-tf
+    inputs:
+      parameters:
+      - name: sha
+    script:
+      image: 245339368379.dkr.ecr.us-east-2.amazonaws.com/builder
+      workingDir: /tmp/git
+      volumeMounts:
+        - name: workdir
+          mountPath: /tmp/git
+      env:
+        - name: TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: github-access-token
+              key: token
+      command: ["sh"]
+      source: |
+        cd k8s_curi
```

Comment:
just a nitpick, but you could set the above `workingDir` to `/tmp/git/k8s_curi` instead